readme.md

```diff
@@ -325,6 +325,7 @@ If you are new to eBPF, you may want to try the links described as "introduction
 - [rbperf](https://github.com/javierhonduco/rbperf) - Sampling profiler and tracer for Ruby.
 - [Hubble](https://github.com/cilium/hubble) - Network, service and security observability for Kubernetes using eBPF.
 - [Caretta](https://github.com/groundcover-com/caretta) - Instant Kubernetes service dependency map generated by eBPF, right to a Grafana instance.
+- [Alaz](https://github.com/ddosify/alaz) - Effortless, Low-Overhead, eBPF-based Kubernetes Monitoring.
```

Comment:
Thanks for the PR!

```suggestion
- [Alaz](https://github.com/ddosify/alaz) - Effortless, low-overhead, eBPF-based Kubernetes monitoring.
```

Please use sentence case for consistency with the other entries.

Can you please rewrite the description into something more objective/informative? “Effortless” and “low-overhead” don't bring much (all eBPF-based projects claim to have low overhead, given they use eBPF). Ideally, it would be good to know what makes it different from other Kubernetes monitoring solutions.